pkg/cli/predict.go

```diff
@@ -120,8 +120,25 @@ func cmdPredict(cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	if err := predictor.Start(os.Stderr); err != nil {
-		return err
+	var stderr bytes.Buffer
+	if err := predictor.Start(&stderr); err != nil {
+		console.Debug(stderr.String())
+
+		if gpus != "" && strings.Contains(err.Error(), docker.ErrMissingDeviceDriver.Error()) {
```

Comment:
Use `errors.Is/As` for more accurate checking?

Reply:
Great idea!